src/main/java/org/sagebionetworks/web/client/PortalGinInjector.java

```diff
@@ -251,240 +251,327 @@
+	SynapseJavascriptClient getSynapseJavascriptClient();
+	SynapseFutureClient getSynapseFutureClient();
+	StuAlert getStuAlert();
+	FilesTabView getFilesTabView();
+	FileTitleBar getFileTitleBar();
+	BasicTitleBar getBasicTitleBar();
+	EntityMetadata getEntityMetadata();
+	FilesBrowser getFilesBrowser();
+	WikiPageWidget getWikiPageWidget();
+	DiscussionThreadListWidget getDiscussionThreadListWidget();
+	ChallengeTabView getChallengeTabView();
+	ChallengeWidget getChallengeWidget();
+	TablesTabView getTablesTabView();
+	QueryTokenProvider getQueryTokenProvider();
+	SettingsPresenter getSettingsPresenter();
+	AccessControlListModalWidget getAccessControlListModalWidget();
+	RenameEntityModalWidget getRenameEntityModalWidget();
+	EditFileMetadataModalWidget getEditFileMetadataModalWidget();
+	EditProjectMetadataModalWidget getEditProjectMetadataModalWidget();
+	EntityFinder getEntityFinder();
+	UploadDialogWidget getUploadDialogWidget();
+	WikiMarkdownEditor getWikiMarkdownEditor();
+	ProvenanceEditorWidget getProvenanceEditorWidget();
+	StorageLocationWidget getStorageLocationWidget();
+	EvaluationEditorModal getEvaluationEditorModal();
+	SelectTeamModal getSelectTeamModal();
+	ApproveUserAccessModal getApproveUserAccessModal();
+	ACTRevokeUserAccessModal getACTRevokeUserAccessModal();
+	ChallengeClientAsync getChallengeClientAsync();
+	UserProfileClientAsync getUserProfileClientAsync();
+	DataAccessClientAsync getDataAccessClientAsync();
+	MultipartFileUploadClientAsync getMultipartFileUploadClientAsync();
+	DiscussionForumClientAsync getDiscussionForumClientAsync();
+	DockerClientAsync getDockerClientAsync();
+	JiraClientAsync getJiraClientAsync();
+	LinkedInServiceAsync getLinkedInServiceAsync();
+	StackConfigServiceAsync getStackConfigServiceAsync();
+	SubscriptionClientAsync getSubscriptionClientAsync();
+	UserAccountServiceAsync getUserAccountServiceAsync();
+	EntityIdCellRendererImpl getEntityIdCellRenderer();
+	UserIdCellRendererImpl getUserIdCellRenderer();
+
+	CreateDataAccessRequestWizard getCreateDataAccessRequestWizard();
+	ManagedACTAccessRequirementWidget getManagedACTAccessRequirementWidget();
+	ACTAccessRequirementWidget getACTAccessRequirementWidget();
+	LockAccessRequirementWidget getLockAccessRequirementWidget();
+	TermsOfUseAccessRequirementWidget getTermsOfUseAccessRequirementWidget();
+	FileHandleWidget getFileHandleWidget();
+	CreateAccessRequirementWizard getCreateAccessRequirementWizard();
+	ProfileCertifiedValidatedWidget getProfileCertifiedValidatedWidget();
+	ACTDataAccessSubmissionWidget getACTDataAccessSubmissionWidget();
+	OpenSubmissionWidget getOpenSubmissionWidget();
+	DateTimeUtils getDateTimeUtils();
+	AccessorGroupWidget getAccessorGroupWidget();
+	AccessRequirementWidget getAccessRequirementWidget();
+	SelfSignAccessRequirementWidget getSelfSignAccessRequirementWidget();
+
+	SubjectWidget getSubjectWidget();
+	TeamMemberRowWidget getTeamMemberRowWidget();
+	RequestBuilderWrapper getRequestBuilder();
+=======

```

Comment:
Doesn't this fail to compile?  (these are usually introduced during a complex merge)

Reply:
Yes, I didn't make the merge correctly, fixing it up now.